Modules/Microsoft365DSC/DSCResources/MSFT_IntuneVPNConfigurationPolicyIOS/MSFT_IntuneVPNConfigurationPolicyIOS.psm1

```diff
@@ -0,0 +1,1340 @@
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $targetedMobileApps,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $Assignments,
+        #endregion
+
+        [Parameter()]
+        [System.String]
+        [ValidateSet('Absent', 'Present')]
+        $Ensure = 'Present',
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $ApplicationSecret,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint,
+
+        [Parameter()]
+        [Switch]
+        $ManagedIdentity,
+
+        [Parameter()]
+        [System.String[]]
+        $AccessTokens,
+
+        [Parameter()]
+        [System.UInt32]
+        $version,
+
+        [Parameter()]
+        [System.String]
+        $loginGroupOrDomain,
+
+        [Parameter()]
+        [System.String]
+        $role,
+
+        [Parameter()]
+        [System.String]
+        $realm,
+
+        [Parameter()]
+        [System.String]
+        $identifier,
+
+        [Parameter()]
+        [System.Boolean]
+        $enablePerApp,
+
+        [Parameter()]
+        [ValidateSet('notConfigured', 'appProxy', 'packetTunnel')]
+        [System.String]
+        $providerType,
+
+        [Parameter()]
+        [System.Boolean]
+        $disableOnDemandUserOverride,
+
+        [Parameter()]
+        [System.Boolean]
+        $disconnectOnIdle,
+
+        [Parameter()]
+        [System.UInt32]
+        $disconnectOnIdleTimerInSeconds,
+
+        [Parameter()]
+        [System.String]
+        $microsoftTunnelSiteId,
+
+        [Parameter()]
+        [System.String]
+        $cloudName,
+
+        [Parameter()]
+        [System.Boolean]
+        $strictEnforcement,
+
+        [Parameter()]
+        [System.String]
+        $userDomain
+    )
+
+    #Ensure the proper dependencies are installed in the current environment.
+    Confirm-M365DSCDependencies
+
+    #region Telemetry
+    $ResourceName = $MyInvocation.MyCommand.ModuleName.Replace('MSFT_', '')
+    $CommandName = $MyInvocation.MyCommand
+    $data = Format-M365DSCTelemetryParameters -ResourceName $ResourceName `
+        -CommandName $CommandName `
+        -Parameters $PSBoundParameters
+    Add-M365DSCTelemetryEvent -Data $data
+    #endregion
+
+    Write-Verbose -Message "Testing configuration of {$id}"
+
+    $CurrentValues = Get-TargetResource @PSBoundParameters
+    $ValuesToCheck = ([Hashtable]$PSBoundParameters).clone()
+
+    if ($CurrentValues.Ensure -ne $Ensure)
+    {
+        Write-Verbose -Message "Test-TargetResource returned $false"
+        return $false
+    }
+    $testResult = $true
+
+    foreach ($key in $PSBoundParameters.Keys)
```

Comment:
Change this comparison chunk with the same one that you also changed in the other resources

Reply:
Fixed it.